### ISSUE TITLE
Update and release OTel agent integration distro 0.13.0

### ISSUE THREAD

**Issue by jasonjkeller** [newrelic/newrelic-opentelemetry-integration-java#47]
This issue is blocked by: https://github.com/newrelic/opentelemetry-exporter-java/issues/145

This project (`newrelic-opentelemetry-integration-java`) needs to be updated to use the OTel dependencies listed below and we need to release version `0.13.0` of this artifact.

The `newrelic-opentelemetry-javaagent` module needs to be updated to use the OTel java agent dependency:
* [io.opentelemetry.javaagent:opentelemetry-javaagent:0.13.1](https://repo1.maven.org/maven2/io/opentelemetry/javaagent/opentelemetry-javaagent/0.13.1/)

The `custom` module needs to be updated to use the New Relic OTel exporter dependencies:
* [com.newrelic.telemetry:opentelemetry-exporters-newrelic:0.13.0](https://repo1.maven.org/maven2/com/newrelic/telemetry/opentelemetry-exporters-newrelic/0.13.0/)
* [com.newrelic.telemetry:opentelemetry-exporters-newrelic-auto:0.13.0](https://repo1.maven.org/maven2/com/newrelic/telemetry/opentelemetry-exporters-newrelic-auto/0.13.0/)

The `smoke-tests` module needs to be updated to use the OTel dependencies:
* [io.opentelemetry:opentelemetry-proto:0.13.1](https://repo1.maven.org/maven2/io/opentelemetry/opentelemetry-proto/0.13.1/)  
* [io.opentelemetry:opentelemetry-api:0.13.1](https://repo1.maven.org/maven2/io/opentelemetry/opentelemetry-api/0.13.1/)

---

**Steps:**
* Update the dependencies used in the `newrelic-opentelemetry-javaagent` and `custom` modules. Both [can be done here](https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/main/build.gradle#L13-L14).
* Update the [dependencies used in](https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/main/smoke-tests/build.gradle#L11-L12) the `smoke-tests` module.
* Update the [CHANGELOG](https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/main/CHANGELOG.md)
* Cut a new [Github release](https://github.com/newrelic/newrelic-opentelemetry-integration-java/releases)

---

**Success criteria:**
* This project (`newrelic-opentelemetry-integration-java`) builds with the OTel artifacts described above
* Before release, verify that the version of this artifact correctly matches the [version to be released](https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/main/gradle.properties#L7)
* A new `newrelic-opentelemetry-integration-java` artifact is released and published to maven
* The [CHANGELOG](https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/main/CHANGELOG.md) is updated
* After release, submit a PR to bump the [snapshot version for the next release](https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/main/gradle.properties#L7)
* All tests pass and the agent distro has been verified to work with the [example application](https://github.com/newrelic/newrelic-opentelemetry-integration-java#example)

**Comment by XiXiaPdx** [comment]
@jasonjkeller  I'm getting the following error when trying to build.  It is for setting the version of the exporter dependency to `0.13.1` 

 Could not GET 'https://oss.jfrog.org/artifactory/oss-snapshot-local/com/newrelic/telemetry/opentelemetry-exporters-newrelic/0.13.1/opentelemetry-exporters-newrelic-0.13.1.pom'. Received status code 409 from server: 
   > Could not resolve com.newrelic.telemetry:opentelemetry-exporters-newrelic-auto:0.13.1.

If I try `0.13.0` instead for the exporter, it works and builds.  [Maven shows 0.13.0](https://search.maven.org/search?q=a:opentelemetry-exporters-newrelic) as latest still.  

Have you seen this error before?

**Comment by jasonjkeller** [comment]
@XiXiaPdx Sorry, I forgot to update this story yesterday based on some findings. Initially the plan was to release version 0.13.0 and 0.13.1 of these artifacts but that changed because the OTel SDK had a borked version that they published.

TL;DR You should use `0.13.0` for both of these artifacts as those are the versions that were published:
* `com.newrelic.telemetry:opentelemetry-exporters-newrelic:0.13.0`
* `com.newrelic.telemetry:opentelemetry-exporters-newrelic-auto:0.13.0`

**Comment by XiXiaPdx** [comment]
Currently blocked by a failing smoke test, this is failing

**Comment by XiXiaPdx** [comment]
Currently blocked by a failing smoke test
https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/1db1cc252c864f3fc221740d96b6eb4c43ff21b3/smoke-tests/src/test/java/com/newrelic/telemetry/BundleTest.java#L47
```
no metrics were reported
java.lang.AssertionError: no metrics were reported
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.newrelic.telemetry.BundleTest.testMetricsAndSpans(BundleTest.java:47)
```
Expected behavior:

Before trying to update dependencies (so running with version 0.12.0), the test passes and the expected metric `processedSpans` is produced. 

Current behavior:

After updating dependencies, the test fails and the logs indictate an empty metric batch: 
```
22:38:23.333 [docker-java-stream--1415956929] INFO  com.newrelic.telemetry.Bundle - STDERR: [opentelemetry.auto.trace 2021-01-28 06:38:23:331 +0000] [IntervalMetricReader-1] DEBUG com.newrelic.telemetry.metrics.MetricBuffer - Creating metric batch.
22:38:23.335 [docker-java-stream--1415956929] INFO  com.newrelic.telemetry.Bundle - STDERR: [opentelemetry.auto.trace 2021-01-28 06:38:23:333 +0000] [Thread-3] DEBUG com.newrelic.telemetry.metrics.MetricBatchSender - Skipped sending of an empty metric batch.
22:38:23.335 [docker-java-stream--1415956929] INFO  com.newrelic.telemetry.Bundle - STDERR: [opentelemetry.auto.trace 2021-01-28 06:38:23:334 +0000] [Thread-3] DEBUG com.newrelic.telemetry.TelemetryClient - Telemetry - MetricBatch - sent
```

I think the issue is with the test and not with the artifact.  After updating dependencies, I did build a 0.13.0 snapshot jar and tested it in spring-petclinic.  I can see the `processedSpans` metric being generated and reported. 

It is a mystery why the test is failing. So far, I’ve tried tripling the timeout here and that didn’t do anything. 

https://github.com/newrelic/newrelic-opentelemetry-integration-java/blob/1db1cc252c864f3fc221740d96b6eb4c43ff21b3/smoke-tests/src/test/java/com/newrelic/telemetry/MockEdge.java#L29

**Comment by jodeev** [comment]
We're not going to make any more updates to the exporter.  We are providing an OTLP endpoint that will support traces, and (shortly) metrics. OR, another alternative is to use the OpenTelemetry Collector.

We were maintaining our exporters until a New Relic OTLP endpoint was ready.  We've reached a point that is close enough to allow customers to migrate to that preferred solution (for use cases where customers do not want to deploy a Collector) rather than adopting this temporary stop-gap.
